processor/src/main/java/org/mapstruct/ap/model/source/SourceMethod.java

```diff
@@ -552,17 +502,6 @@ public boolean isStatic() {
         return executable.getModifiers().contains( Modifier.STATIC );
     }
 
-    private void increase(Type key, Map<Type, Integer> test) {
```

Comment:
Apparently I'm not working enough on the code base. You guys are removing stuff which I didn't even know it was in there ;)